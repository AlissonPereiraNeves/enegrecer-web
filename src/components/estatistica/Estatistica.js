import React from 'react';
import './css/estatistica.css';

const Estatistica = () => (
  <div id="estatistica">

    <div id="nosso-objetivo">
      <div className="retangular-bloco" />
      <h6>Nosso Objetivo </h6>
      <p>Buscamos dar visibilidade para crimes contra negros a partir de uma
      base de dados acessível, transparente e atualizada. </p>
    </div>

    <div id="rectangle1">
      <h3 className="numero">2345 </h3>
      <h6 className="categoria">Racismo </h6>
    </div>

    <div id="rectangle2">
      <div id="rectangle21">
        <div id="amarelo">
          <h3 className="numero">2345 </h3>
          <h6 className="categoria">Injúria Racial</h6>
        </div>
        <div id="verde">
          <h3 className="numero">60% </h3>
          <h6 className="categoria">Vítimas Mulheres</h6>
        </div>
      </div>

      <div id="rectangle22">
        <div id="cinza">
          <h3 className="numero">2345 </h3>
          <h6 className="categoria">Vítima até 26 anos</h6>
        </div>
        <div id="cinzaClaro">
          <h3 className="numero">346 </h3>
          <h6 className="categoria">Total de Denúncia</h6>
        </div>
      </div>

    </div>
  </div>
);

export default Estatistica;
