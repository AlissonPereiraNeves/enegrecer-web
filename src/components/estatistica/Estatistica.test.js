import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Estatistica from './Estatistica';

describe('Estatistica', () => {
  const wrapper = shallow(<Estatistica />);

  it('possui a mesma estrutura', () => {
    const tree = toJson(wrapper);
    expect(tree).toMatchSnapshot();
  });
});
