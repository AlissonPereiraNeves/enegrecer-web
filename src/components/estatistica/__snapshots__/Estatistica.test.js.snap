// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Estatistica possui a mesma estrutura 1`] = `
<div
  id="estatistica"
>
  <div
    id="nosso-objetivo"
  >
    <div
      className="retangular-bloco"
    />
    <h6>
      Nosso Objetivo 
    </h6>
    <p>
      Buscamos dar visibilidade para crimes contra negros a partir de uma base de dados acess√≠vel, transparente e atualizada. 
    </p>
  </div>
  <Racismo
    key="racismo"
  />
  <div
    id="rectangle2"
  >
    <div
      id="rectangle21"
    >
      <Injuria
        key="injuria"
      />
      <VitimasMulheres
        key="vitimasMulheres"
      />
    </div>
    <div
      id="rectangle22"
    >
      <VitimasAte29Anos
        key="vitimasAte29Anos"
      />
      <TotalDeDenuncia
        key="totalDeDenuncia"
      />
    </div>
  </div>
</div>
`;
